src/cascadia/TerminalApp/TabHeaderControl.xaml

```diff
@@ -0,0 +1,31 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under
+the MIT License. See LICENSE in the project root for license information. -->
+<UserControl
+    x:Class="TerminalApp.TabHeaderControl"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:TerminalApp"
+    xmlns:mux="using:Microsoft.UI.Xaml.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <StackPanel x:Name="HeaderStackPanel"
+                Orientation="Horizontal">
+        <FontIcon x:Name="HeaderZoomIcon"
+                  FontFamily="Segoe MDL2 Assets"
+                  Visibility="Collapsed"
```

Comment:
you could, in theory, `{x:Bind IsPaneZoomed}` and make it an observable boolean :smile:

Reply:
(it is safe to bind a boolean to a visibility! surprise!)